packages/api/src/routes/medical/patient.ts

```diff
@@ -533,4 +537,65 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /patient/:id/facilities
+ *
+ * Assigns a patient to multiple facilities.
```

Comment:
ditto: "set" vs "assign" according to what we decide on the other thread in the SDK